### PR TITLE
Fix settings starting with server being treat as backend server

### PR DESCRIPTION
**Pull request by alisson276**
This change fix settings like `server-state-file` and `server-state-base` global settings be treat as ServerLine.

### CODE REVIEW

**Comment by imjoey** [comment]
@alisson276 Thanks for your contribution. Since there's something wrong with the travis CI configuration, Could you please rebase the code to contain the latest code of CI failure solution? Thank you.

File: pyhaproxy/test.py [hunk 1]
```diff
@@ -153,47 +153,47 @@ def teardown(self):
         pass
 
     def test_parse_global_section(self):
-        print self.configration
```
Comment (imjoey) [inline pyhaproxy/test.py:156]: @alisson276 thanks for the update. Maybe we need to add an extra `from __future__ import print_function` at the top line of the file, which guarantees us using print as a function.

Comment (alisson276) [reply pyhaproxy/test.py:156]: @imjoey , it's done!

**Comment by imjoey** [comment]
LGTM. @alisson276  Thank you very much.

**Pull request merged**